Tasks/DownloadPackageV1/multifilepackage.ts

```diff
@@ -81,7 +82,7 @@ export class MultiFilePackage extends Package {
                         if (filteredFileList.has(fileMetadatas[i].name)) {
                             const fileMetadata = fileMetadatas[i];
                             pkgFileUrlPromises.push(
-                                fileMetadata.protocolMetadata.data.storageId != null
+                                (fileMetadata.protocolMetadata.data.StorageId != null || fileMetadata.protocolMetadata.data.storageId != null)
```

Comment:
nit: I usually prefer to create variable if property you're accessing is deeply nested, seems more readable to me
```
let protocolData = fileMetadata.protocolMetadata.data;
protocolData.StorageId != null || protocolData.storageId != null
```